app/views/user_sessions/_new.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container">
 <%= form_for :user_session, :as => :user_session, :url => user_sessions_url + "?return_to=" + request.fullpath, :html => {:class => "form col-md-6 offset-3"} do |f| %>
```

Comment:
The login modal also redirects to publiclab.org. Should we change this to `:url => "/user_sessions?return_to=" + request.fullpath` just like it was changed for signup?

Reply:
I think we should create a separate pr for that.
As login modal works perfectly fine on the main production branch.
Can you create a separate PR for it?
Thanks
Let's try to keep this PR more convergent to the sign up functionality.
Thanks

Reply:
I can do that :+1:

Reply:
Great. Refer #4324